docs/docs/pyproject.md

```diff
@@ -133,6 +133,59 @@ packages = [
     Poetry is clever enough to detect Python subpackages.
 
     Thus, you only have to specify the directory where your root package resides.
+    
+## data_files
+
+A list of files to be installed using the [data_files](https://docs.python.org/2/distutils/setupscript.html#installing-additional-files)
```

Comment:
Lets link to the [Python 3 version](https://docs.python.org/3/distutils/setupscript.html#installing-additional-files)?